archlinux/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Danil Osherov <shindo@yandex-team.ru>
+
+pkgname=blackhole
+pkgdesc="Yet another C++ logging library."
+pkgver=0.2.3
+pkgrel=1
+arch=('x86_64')
+url=('https://github.com/3Hren/blackhole')
+license=('LGPL3')
+depends=('boost>=1.46')
+makedepends=('git' 'cmake')
+source=("$pkgname"::"git://github.com/shindo/blackhole.git#tag=v$pkgver")
```

Comment:
Nope, it was on purpose. I use git tags to checkout source code for specific version (format is just like yours - "v(version)", e.g. v0.2.3).
It wasn't possible to add tag to your repository (I haven't necessary permissions), so I used mine. 

I'll change it back to yours repository, just don't forget to update v0.2.3 tag.